src/components/structures/RightPanel.js

```diff
@@ -187,19 +186,22 @@ export default class RightPanel extends React.Component {
             panel = <GroupMemberList groupId={this.props.groupId} key={this.props.groupId} />;
         } else if (this.state.phase === RIGHT_PANEL_PHASES.GroupRoomList) {
             panel = <GroupRoomList groupId={this.props.groupId} key={this.props.groupId} />;
-        } else if (this.state.phase === RIGHT_PANEL_PHASES.RoomMemberInfo) {
+        } else if (this.state.phase === RIGHT_PANEL_PHASES.RoomMemberInfo ||
+            this.state.phase === RIGHT_PANEL_PHASES.EncryptionPanel) {
```

Comment:
Maybe indent this more like:

```
} else if (
    this.state.phase === RIGHT_PANEL_PHASES.RoomMemberInfo ||
    this.state.phase === RIGHT_PANEL_PHASES.EncryptionPanel
) {
```

Mainly I find it odd to see the conditional's second line at the same indent as the code inside the block with no other line between line.